kernel/cooking.ml

```diff
@@ -315,7 +315,7 @@ let cook_constant env info cb =
 (********************************)
 (* Discharging mutual inductive *)
 
-let abstract_rel_context cache info ctx =
+let cook_rel_context cache info ctx =
```

Comment:
IMO I prefer `abstract` to `cook` but it's not a strong preference.

Reply:
I like `abstract` also but `cook` is used many times already.

Reply:
>I like abstract also but cook is used many times already.

Naming isn't a democracy, we can be tyrannical here